src/listener.js

```diff
@@ -1,183 +1,234 @@
 'use strict'
 
-const LP = require('./rpc/lp')
-const pull = require('pull-stream/pull')
-const handshake = require('pull-handshake')
-const { JoinInit, JoinChallenge, JoinChallengeSolution, JoinVerify, Discovery, DialRequest, DialResponse, ErrorTranslations } = require('./rpc/proto')
-
-const prom = (f) => new Promise((resolve, reject) => f((err, res) => err ? reject(err) : resolve(res)))
+const debug = require('debug')
+const log = debug('libp2p:stardust:listener')
+log.error = debug('libp2p:stardust:listener:error')
 
-const sha5 = (data) => crypto.createHash('sha512').update(data).digest()
+const EventEmitter = require('events')
```

Comment:
We should be consistent with the imports, `src/index.js` is importing this way:
```suggestion
const { EventEmitter } = require('events')
```